src/pages/Home/sections/Contact/components/Map/map.scss

```diff
@@ -8,48 +10,24 @@
   display: none;
 }
 
-.mapToolTip,
-.reCenter {
-  position: absolute;
-  bottom: 10px;
-  z-index: 400;
-  font-size: 17px;
-  font-weight: 300;
-  line-height: 0.5;
-  font-family: monospace, "sans-serif";
-  cursor: default;
-}
-
-.mapToolTip {
-  width: 230px;
-  padding: 1.5rem;
-  left: 10px;
-  background: rgba(29, 24, 24, 0.9);
-  color: #fff;
-  border-radius: 5px;
-}
-
-.mapToolTip a {
-  cursor: pointer;
-}
-
 .leaflet-popup-content-wrapper {
-  background-color: rgba(29, 24, 24, 0.9);
-  color: white;
+  background-color: colors.$black_0-8;
+  color: colors.$white;
 }
+
 .leaflet-popup-tip {
-  background-color: rgba(29, 24, 24, 0.9);
+  background-color: colors.$black_0-8;
```

Comment:
This could be made better too rather than using black_0-8 use proper names to these. you colour google names of these colours and add them.